src/loose/index.js

```diff
@@ -41,9 +41,7 @@ defaultOptions.tabSize = 4
 
 // eslint-disable-next-line camelcase
 export function parse_dammit(input, options) {
-  let p = new LooseParser(input, options)
-  p.next()
-  return p.parseTopLevel()
```

Comment:
@marijnh You do see how having two methods named `parseTopLevel` that do two different things is confusing though right?